onionperf/onionperf

```diff
@@ -116,12 +116,18 @@ def main():
         default="-")
 
     monitor_parser.add_argument('-e', '--event',
```

Comment:
Can we change this to '--events' (make it plural) to match the new '--custom-events' option?